### DISCUSSION TITLE
How to actually import in JS frameworks ?

### DISCUSSION THREAD

**Discussion by KaKi87** [jarek-foksa/xel#116] [Q&A] [answered]
Hello,

The FAQ states that :
> Xel is framework-agnostic. Any framework that [supports custom elements](https://custom-elements-everywhere.com/) should be compatible with Xel.

But, how to actually import it in JS frameworks ? For example, Vue ?

Thanks

**Comment by jarek-foksa** [comment, marked as answer]
You can import Xel just like any other regular ES module, e.g. 

```js
import "/node_modules/xel/xel.js";
```

or

```js
await import("/node_modules/xel/xel.js");
```

**Reply by KaKi87** [reply]
Thanks, I also needed [this](https://vuejs.org/guide/extras/web-components.html#using-custom-elements-in-vue).
